extra/o11y_platforms.md

```diff
@@ -0,0 +1,49 @@
+# Integrating with Observability Platforms
+
+While most monitoring and observability platforms should be able to utilize the OpenMetrics standard, some platforms may require some form of minor config or development to integrate. This is normal for any device providing metrics, not just Pure Storage.
+
+Pure Storage are working with observability platform vendors to ensure products work out of the box.
+
+## Datadog
+
+https://app.datadoghq.com/integrations/purefa
+
+This check monitors the Pure Storage FlashArray through the Datadog Agent and the Pure Storage OpenMetrics exporter.
+
+The integration can provide performance data at the array, host, volume and pod level, as well as high-level capacity and configuration information.
+
+You can monitor multiple FlashArrays and aggregate these into a single dashboard, or group them together by customer defined environment.
+
+<br>
+<img src="images/purefa_datadog_integration.png" width="50%" height="50%">
+<br>
+
+## Dynatrace
+
+https://www.dynatrace.com/hub/detail/pure-storage-flasharray
+
+Dynatrace extension 2.0 enables automates metrics ingestion and performance data corrolation. With 2x dashboards iuncluded in the bundle, users can drill down through array configurations items and monitor their fleet of arrays providing an overview of capacity utilization, data reduction and Purity version.
```

Comment:
recheck grammar, syntax and spelling

Reply:
Thanks for spotting Simon - now resolved

Reply:
Not resolved

Reply:
Sorry Simon - check latest commit, fixed.